Modules/IO/NRRD/test/itkNrrdImageIOTest.h

```diff
@@ -99,6 +100,20 @@ itkNrrdImageIOTestReadWriteTest(std::string fn, unsigned int size, std::string i
   {
     // Generate a random image.
     image = itkNrrdImageIOTestGenerateRandomImage<TPixelType, VImageDimension>(size);
+
+    // add custom metadata
+    itk::MetaDataDictionary dictionary;
+
+    itk::EncapsulateMetaData(dictionary, "ASimpleString", std::string("a string"));
+    itk::EncapsulateMetaData(dictionary, "ASimpleFloat", 1.2f);
+    itk::EncapsulateMetaData(dictionary, "ASimpleDouble", 2.3d);
```

Comment:
This suffix _**d**_ is supported only by some versions of _GCC_. Not in standard, AFAIK.

```cpp
#include <iostream>
int main(int, char**)
{
	double x = 0.123456d;
	std::cout << x << std::endl;
	return 0;
}
```
```

r@deb2:~$ g++ -Wpedantic test.cpp 
qq.cpp:4:20: warning: suffix for double constant is a GCC extension
    4 |         double x = 0.123456d;
      |                    ^~~~~~~~~

```